<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormTeachers</class>
 <widget class="QDialog" name="FormTeachers">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>885</width>
    <height>543</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>461</width>
     <height>521</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>10</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="button_Add">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="text">
       <string>Dodaj Nauczyciela</string>
      </property>
      <property name="icon">
       <iconset resource="zasoby.qrc">
        <normaloff>:/z/icons/addteacher.png</normaloff>:/z/icons/addteacher.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="button_Edit">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Edytuj</string>
      </property>
      <property name="icon">
       <iconset resource="zasoby.qrc">
        <normaloff>:/z/icons/editteacher.png</normaloff>:/z/icons/editteacher.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="button_Delete">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Usuń Nauczyciela</string>
      </property>
      <property name="icon">
       <iconset resource="zasoby.qrc">
        <normaloff>:/z/icons/delteacher.png</normaloff>:/z/icons/delteacher.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>100</y>
     <width>381</width>
     <height>431</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="2">
     <widget class="QLineEdit" name="lE_Email">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Nazwa użytkownika</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Specjalizacja</string>
      </property>
     </widget>
    </item>
    <item row="5" column="2">
     <widget class="QComboBox" name="comboBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>E-mail</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Imię</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Hasło</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Nazwisko</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLineEdit" name="lE_Surname">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="lE_Password">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QLineEdit" name="lE_Name">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLineEdit" name="lE_UserName">
      <property name="enabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>60</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QPushButton" name="button_Confirm">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="zasoby.qrc">
        <normaloff>:/z/icons/right-tick.png</normaloff>:/z/icons/right-tick.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="button_Update">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="zasoby.qrc">
        <normaloff>:/z/icons/update-arrow.png</normaloff>:/z/icons/update-arrow.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="button_Canel">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="zasoby.qrc">
        <normaloff>:/z/icons/gross-dark-cross.png</normaloff>:/z/icons/gross-dark-cross.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="button_ShowPassword">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="zasoby.qrc">
        <normaloff>:/z/icons/magnifier-with-an-eye.png</normaloff>:/z/icons/magnifier-with-an-eye.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="zasoby.qrc"/>
 </resources>
 <connections/>
</ui>
